fix(sdk-core): fix createTssBitGoKeyFromOvcShares method

alebusse opened the pull request:
changed the method response to match with OVC expected JSON, wrapping the response in wallet: {...} and renaming the walletgpgkeys field

WP-152

<!--
# Please be aware of the following when making your pull request:

## Description

Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue Number

Internal Users - Please include the related internal tracking number (e.g. BG-000000).
External Users - Please link to any relevant github issues as necessary.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code compiles correctly for both Node and Browser environments
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [ ] The ticket or github issue was included in the commit message as a reference
- [ ] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-->

OttoAllmendinger reviewed (changes requested):
please submit whitespace changes separately

I thought we had a prettier set up for this?

zahin-mohammad commented on modules/sdk-core/src/bitgo/keychain/ovcJsonCodec.ts:
```diff
@@ -45,26 +45,28 @@ export const OvcToBitGoJSON = t.strict(
-          // BitGo to Backup (OVC-2)
-          2: t.type({
-            bitgoToOvcShare: OvcShare,
+export const BitGoToOvcJSON = t.strict(
```
you can create a type from this.

zahin-mohammad commented on modules/sdk-core/src/bitgo/keychain/keychains.ts:
```diff
@@ -374,37 +374,39 @@ export class Keychains implements IKeychains {
 
     // Create JSON data with platform shares for OVC-1 and OVC-2
     const bitgoToOvcOutput = {
```
you can add a type to this from the codec type.

alebusse committed:
fix(sdk-core): fix createTssBitGoKeyFromOvcShares method

changed the method response to match with OVC expected JSON, wrapping the response in wallet: {...}
and renaming the walletgpgkeys field

WP-152
